docs/src/main/sphinx/admin/event-listeners-openlineage.md

```diff
@@ -157,6 +157,12 @@ event-listener.config-files=etc/openlineage-event-listener.properties,...
     - Custom namespace to be used for Job `namespace` attribute. If blank will
       default to Dataset Namespace.
     - None.
+*
+    - openlineage-event-listener.job.nameFormat
+    - Custom namespace to be used for Job `name` attribute. Support substitution
+      variables: `$QUERY_ID`, `$USER`, `$SOURCE`.
+      If blank will default to `$QUERY_ID`.
```

Comment:
can you give more thorough documentation of what would be the accepted variations here and how they would be resolved by trino? I think examples from tests would be enough, just added to the docs for users.

Reply:
Added 2 examples, please check.